vendor/arbiter/arbiter.cpp

```diff
@@ -2720,6 +2749,25 @@ std::string AZ::Config::extractStorageAccessKey(
     return "";
 }
 
+std::string AZ::Config::extractSasToken(const std::string s)
```

Comment:
Is there a reason this isn't a reference, given that it's already const? Some threading thing?